### ISSUE TITLE
Let users annotate their contributions

### ISSUE THREAD

**Issue by adklempner** [AurelienLourot/ghuser.io#144]
Hi, awesome project! I've been playing around with a similar idea because I thought GitHub profiles are lacking and can do so, so much more. Having worked almost exclusively on open source projects, I don't like LinkedIn at all and find no value in it. However, just telling a potential employer to "look through my GitHub" is unprofessional.

A simple addition that could allow one to use their profile as more of a personal portfolio/resume/CV is the ability to annotate their contributions with markdown. Something like this:

![md annotations](https://user-images.githubusercontent.com/22138672/45348297-ee8ce600-b5cb-11e8-9790-a15150d18dad.gif)

Any claims made by the annotator can be cross-referenced with your metrics that are generated from GitHub (which acts as a trusted third party in this case)

Would love to help out/explore this further with you 🙂

**Comment by AurelienLourot** [comment]
Good idea, thanks! I can't pick it up right now because I have to focus on #143 (1700+ people wanted a profile yesterday and we're far away from being able to deliver). 

But if you can code and have time, we're more than happy to get pull requests :)
